crate/src/lib.rs

```diff
@@ -21,8 +21,7 @@ use serde_json;
 use std::{convert::identity, fmt};
 use Visibility::*;
 
-const SEED_VERSION: &str = "0.5.1";
-const SEED_VERSION_DATE: &str = "Dec 28, 2019";
+const SEED_VERSION: &str = "0.6.0 (February 1, 2020)";
```

Comment:
I've split it into two variables in this PR https://github.com/seed-rs/seed-rs.org/pull/23 for more flexible styling